Using-browsec-for-securing-your-connection-to-fcc.md

```diff
@@ -0,0 +1,89 @@
+
+<img src='./images/Using-browsec-for-securing-your-connection-to-fcc/Browsec-Chrome.png' alt="Image for 'Browsec on Google Chrome WebStore'">
+
+
+##### Step 2 : Clear your browser cookies and cache (optional).
+It's a good thing if you clear your browser cache for the first time you are going to use browsec, so that your browser loads all the files from scratch.
+
+##### Step 3 : Restart your browser, and visit [FreeCodeCamp.com](http://freecodecamp.com)
+Just close your browser, and restart it. Check the browsec extention, for your desired endpoint location.
+
+### For Mozilla Firefox : 
+
+Download a portable Firefox version, with bundled add-on, from the [Browsec website](https://browsec.com/en/dashboard/main).
+
+<img src='./images/Using-browsec-for-securing-your-connection-to-fcc/Browsec-Firefox.png' alt="Image for 'Browsec on Google Chrome WebStore'">
+
+
+That's it! Happy Coding! If this works then do let us know in the [Help Chat](https://gitter.im/FreeCodeCamp/Help)
+
+## FAQs
+
+### How do I know, if I am facing a "man in the middle" attack?
+When you visit the FCC website, or do the challenges, if you look at your browser's Developer console (Press F12 > Console tab), you should see similar errors like below.
+
+<img src="./images/Using-browsec-for-securing-your-connection-to-fcc/Error_1.png" alt="Image for 'Error'">
+
+<img src="./images/Using-browsec-for-securing-your-connection-to-fcc/Error_2.png" alt="Image for 'Error'">
+
+<img src="./images/Using-browsec-for-securing-your-connection-to-fcc/Error_3.png" alt="Image for 'Error'">
+
+These are classic cases of the issue.
+
+### Wait a minute, Can I use any alternate mechanism, to Browsec?
+Yes, why not, you can use any available VPN clients in the market, but keep in mind that Browsec is free and works just great!
+
+### Hey what about other browsers, Internet Explorer, Safari, e.t.c.? 
+Hmm, go for any VPN add-on that you can find for these browsers, [Tor](https://www.torproject.org/), is one such client, but they come with paid subscriptions, you can basically use any anonymizer that you want, however Chrome and Browsec are tried and tested and have worked for most campers in the past.
+
+### Can I use the anonymizer for using websites other than FCC?
+Of course yes. Why not? However remember that this does not make you invisible to law, so be careful of what you are doing! :wink:
+
+### What if this doesn't work for me?
+Please let us know in the [Help Chat](https://gitter.im/FreeCodeCamp/Help), we will try our best to find a workaround.
+
+---
+
+##### *Disclaimer*
+
+***FreeCodeCamp does NOT endorse any of the softwares mentioned in this article, please use them at your own discretion. Some VPN clients may track your activity, also there could be side-effects such as slow speeds or delay in page loads e.t.c.***
```

Comment:
It would be "etc", but I suggest just remove `e.t.c.` entirely.  Just put a period and be done with it.
> such as slow speeds or delays in page loads.